packages/flutter_test/lib/src/goldens.dart

```diff
@@ -65,17 +70,93 @@ abstract class GoldenFileComparator {
         .join() + '.' + version.toString() + extension
     );
   }
+
+  /// Returns a [ComparisonResult] to describe the pixel differential of the
+  /// [test] and [master] image bytes provided.
+  static ComparisonResult compareLists<T>(List<int> test, List<int> master) {
+    if (identical(test, master))
+      return ComparisonResult(passed: true);
+
+    if (test == null
+      || master == null
```

Comment:
This is odd formatting: we use the form with the operators at the beginning for return values because it happens to line up nicely with the word "return", but for `if` conditionals we usually put the operators at the end of the line.